prototypes/alleleFrequencies/renderAlleles.py

```diff
@@ -0,0 +1,113 @@
+import vcfAlleleLibrary
+import matplotlib.pyplot as plt
+import numpy as np
+
+"""Given a list of ExAC alleles, a bar chart of allele frequencies is plotted, with 
+subpopulations on the x axis and frequencies on the y axis"""
+
+
+def plotAlleles(alleleList):
+    """Given an alleleList, will produce 2 bar charts rendering allele freq
+    data"""
+    figureNum = 0 #will allow "figure 1" and "figure 2" to be made
+    for allele in alleleList: #allele is an object with attributes
+        figureNum += 1
+        if len(allele.pops) == 5:
```

Comment:
This mechanism looks fragile.  It will work for now, but will break when the number of subpopulations changes for either allele list (and gnomAD, the successor to ExAC, contains additional subpopulations).  A better approach would be to pass in a parameter that indicates which allele set you're working wiht.